backends/ebpf/target.h

```diff
@@ -238,6 +238,11 @@ class P4TCTarget : public KernelSamplesTarget {
         }
         return "HOST"_cs;
     }
+
+    bool isPrimitiveByteAligned(int width) const {
+        return (width <= 8 || width <= 16 || (width > 24 && width <= 32) ||
```

Comment:
Looks width <= 8 check is redundant
```suggestion
        return (width <= 16 || (width > 24 && width <= 32) ||
```